popup.js

```diff
@@ -44,70 +47,136 @@ function refreshBalance(value) {
+            let txType = document.getElementById("tx-type-" + divNumber);
+
+            txInfo.innerHTML = ((amount * 100) / 100).toFixed(2) + " to ";
+
+            if (account != "") {
+                txInfo.innerHTML += " " + account;
             }
 
             txInfo.innerHTML += "<br>";
 
-            if (latestTransactions[i].address !== undefined) {
-                txInfo.innerHTML += latestTransactions[i].address;
-            } else if (latestTransactions[i].category == "send") {
-                if (latestTransactions[i].amount === -0.1 && latestTransactions[i].fee === -0.1) {
+            if (address !== undefined) {
+                txInfo.innerHTML += address;
+            } else if (category == "send") {
+                if (amount === -0.1 && fee === -0.1) {
                     txInfo.innerHTML += "BlockchainDNS Registration";
-                    
+
                     txType.innerHTML = "REG";
                     txType.style.color = "rgb(30, 118, 210)";
 
                     continue;
-                } else if (latestTransactions[i].amount === -0.005 && latestTransactions[i].fee === -0.005) {
+                } else if (amount === -0.005 && fee === -0.005) {
                     txInfo.innerHTML += "BlockchainDNS Update";
-                    
+
                     txType.innerHTML = "UPD";
                     txType.style.color = "rgb(30, 118, 210)";
 
                     continue;
                 }
             }
 
-            if (latestTransactions[i].category == "generate" || latestTransactions[i].category == "receive") {
-                txType.innerHTML = "IN";
-                txType.style.color = "rgb(50, 205, 50)";
-            } else if (latestTransactions[i].category === "send") {
-                txType.innerHTML = "OUT";
-                txType.style.color = "rgb(255, 0, 0)";
-            } else if (latestTransactions[i].category == "immature") {
-                txType.innerHTML = "IMM";
-                txType.style.color = "rgb(192, 192, 192)";
-            } else if (latestTransactions[i].category == "orphan") {
-                txType.innerHTML = "OPH";
-                txType.style.color = "rgb(192, 192, 192)";
+            switch (category) {
+                case "generate": case "receive":
+                    txType.innerHTML = "&#8592;";
+                    txType.style.color = "rgb(50, 205, 50)";
+                    break;
+                case "send":
+                    if (isToSelf(i, address, amount)) {
+                        txType.innerHTML = "&#8651;";
+                        txType.style.color = "rgb(138,43,226)";
+                        i--;
+                    } else {
+                        txType.innerHTML = "&#8594;";
+                        txType.style.color = "rgb(255, 0, 0)";
+                    }
+                    break;
+                case "immature":
+                    txType.innerHTML = "IMM";

```

Comment:
due to the increased font size this appears too big and overlaps with the tx info